frontend/src/services/applicationService.js

```diff
@@ -98,6 +98,32 @@ export default {
     }
   },
 
+  async getSupplementaryApplicationsForForm(applicationId) {
+    try {
+      if (!applicationId) return
+      const url = `${ApiRoutes.SUPPLEMENTARY_APPLICATIONS}/${applicationId}`
+
+      const response = await ApiService.apiAxios.get(url)
+      //clean up the fields I don't need
```

Comment:
Would it make more sense to just include the fields you do need? What if I new field is added? Will it cause issues if not deleted here?

Reply:
nope -it was leftover from the other code - it's gone now

Reply:
Discussed with Jen and there are more fields retained than deleted so the current approach makes sense.